spec/magma_commands_spec.rb

```diff
@@ -0,0 +1,300 @@
+describe 'Magma Commands' do
+  describe Magma::LoadProject do
+    let(:json_file) { './spec/fixtures/template.json' }
+
+    subject(:load_project) { described_class.new.execute('project_name', json_file) }
+
+    let(:attributes) { Magma.instance.db[:attributes] }
+
+    it 'loads attributes into the database' do
+      json_attributes = JSON.parse(JSON.parse(File.open(json_file).read))["models"]["aspect"]["template"]["attributes"]
```

Comment:
I find this double-parse confusing, can you clarify what this is doing?

Reply:
I meant to point this out as a weird thing that is happening. Unsure of why I need to double parse, was hoping someone had knowledge of it than I would? Maybe the file being stored in the fixtures is written incorrectly?